x-pack/plugins/security_solution/public/management/components/console/components/command_execution_output.tsx

```diff
@@ -62,6 +65,30 @@ export const CommandExecutionOutput = memo<CommandExecutionOutputProps>(
       [dispatch, id]
     );
 
+    // keep track if this becomes a long running command
+    useEffect(() => {
+      let timeoutId: number;
+
+      if (isRunning && !isLongRunningCommand) {
+        const elapsedSeconds = moment().diff(moment(enteredAt), 'seconds');
+
+        if (elapsedSeconds >= 15) {
+          setIsLongRunningCommand(true);
+          return;
+        }
+
+        timeoutId = setTimeout(() => {
+          setIsLongRunningCommand(true);
+        }, (15 - elapsedSeconds) * 1000);
+      }
```

Comment:
So here the `elapsedSeconds` never increments or is always `0`. Basically, the timeout is set to execute after 15 seconds and sets the `isLongRunningCommand` to `true`. We're not actually using the time difference here.

Reply:
Not sure what you mean.

Let me explain this logic around `elapsedSeconds`. You can enter a command and then close the Responder window... When we re-open that same responder window, we need to determine how much time has elapsed since the command was entered so that we can correctly calculate the `setTimeout()` ms. 

Example:

- I enter a command at `:00 s` and immediately close the console
- I reopen the console just 5s later.
- the `setTimeout()` should now only wait `10s` since 5s has already elapsed...  so the `(15 - elapsedSeconds)` does that.
- This logic here will only be triggered if `15s` has not yet elapsed

Reply:
Oh, I see. When the command is executed and the console stays open beyond 15 seconds the `elapsedSeconds` is always 0 and thus the `elapsedSeconds >= 15` block does not execute is what I meant. I didn't test it by closing and then re-opening the console, in which case the block does execute and it works as expected.